docs/ref/models/querysets.txt

```diff
@@ -1750,6 +1750,43 @@ See the :doc:`/topics/db/sql` for more information.
   filtering. As such, it should generally be called from the ``Manager`` or
   from a fresh ``QuerySet`` instance.
 
+
+Operators that return new ``QuerySet``\s
+----------------------------------------
+
+AND (``&``)
+~~~~~~~~~~~
+
+Combines two ``QuerySet``\s by merging their filters, equivalent to
+the ``AND`` operator.
+
+All of the following three expressions are equivalent::
+
+  >>> Model.objects.filter(x=1) & Model.objects.filter(y=1)
+  >>> Model.objects.filter(x=1, y=2)
+  >>> Model.objects.filter(Q(x=1) & Q(y=2))
```

Comment:
The `>>>` syntax is for interactive sessions - which would have a result. I think the example should be without it, see e.g. `only()`

Not sure the `Q` example is adding much either, Q objects are mentioned later.